### PR TITLE
Feature/install primereact

### PR DESCRIPTION
**Pull request by ulrokx**
Installing Primereact so that we can use their prebuilt components. Custom styling is available and will be the preferred route so that we can adhere to brand assets.

### CODE REVIEW

**Comment by ShawnAviles** [comment]
Installation is not done. The components are not visible in the app yet. Further development is still needed.

**Comment by ShawnAviles** [comment]
> Installation is not done. The components are not visible in the app yet. Further development is still needed.

All good now. Just needed to add some sample custom styling for the component. Extracted into a separate `designSystem.tsx` file so we can add on more styles from there.

File: src/styles/designSystem.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+interface HeaderProps {
+  toggleable?: boolean;
+  // add other properties as needed
+}
+
+// Customize PrimeReact components
+const designSystem = {
+  panel: {
+    header: ({ props }: { props: HeaderProps }) => ({
+      className: classNames(
```
Comment (ulrokx) [inline src/styles/designSystem.tsx:12]: Wow this is actually pretty nice, way better than one long string with template literals. TIL

**Review by ulrokx — COMMENTED** [review]
👌

**Pull request merged**